requirements/edx/testing.txt

```diff
@@ -202,7 +202,7 @@ piexif==1.1.3             # via -r requirements/edx/base.txt
 pillow==7.2.0             # via -r requirements/edx/base.txt, edx-enterprise, edx-organizations
 pluggy==0.13.1            # via -r requirements/edx/coverage.txt, diff-cover, pytest, tox
 polib==1.1.0              # via -r requirements/edx/base.txt, -r requirements/edx/testing.in, edx-i18n-tools
-psutil==5.7.2             # via -r requirements/edx/base.txt, edx-django-utils, pytest-xdist
```

Comment:
psutil is now an optional dependency of pytest-xdist, but we still want the functionality it's used for (more accurate count of available CPUs for parallelizing tests).  Our dependency on this should switch in `test.in` from `pytest-xdist` to `pytest-xdist[psutil]`.

Reply:
Sure @jmbowman. I'll open a JIRA ticket for it.